ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,215 @@
+                            <title type="complete" ref="LIT1990Mawase">Mawāśǝʾǝt, Antiphonary for feasts and funeral service
+                                
+                            </title>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="60">209</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm">375</height>
+                                    <width unit="mm">320</width>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="46">   
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc>A. d'Abbadie desribes the handwriting as <foreign xml:lang="fr">'fine et jolie'</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#209v" ></locus>
+                                    <desc type="GuestText">According to Conti Rossini 
+                                        , there is a <foreign xml:lang="fr">'note en amharique, contenant des indications testimoniales sur un achat de terre non spécifié'</foreign>
+                                        <bibl>
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">199-120</citedRange>
+                                        </bibl> 
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc cert="unknown">
+                            <binding contemporary="Ethiopian">                  
+                                <decoNote xml:id="b1" type="Other">A. d'Abbadie desribes the binding as <foreign xml:lang="fr">'belle reliure peu usée'.</foreign>. </decoNote>                  
```

Comment:
It should be "describes"

Maybe he describes it "as a…", because "reliure" is in the description :)